we_count/README.md

```diff
@@ -0,0 +1,22 @@
+# we_count
+
+> We Count website
+
+## Build Setup
+
+``` bash
+# install dependencies
+$ npm run install
```

Comment:
For the standard npm scripts you don't need to call `run` before the script name. https://docs.npmjs.com/misc/scripts